docs/layouts/shortcodes/build_table.html

```diff
@@ -0,0 +1,42 @@
+<style type="text/css">
+  .tg  {border-collapse:collapse;border-spacing:0;}
+  .tg td{border-color:black;border-style:solid;border-width:1px;font-family:Arial, sans-serif;font-size:14px;
+    overflow:hidden;padding:10px 5px;word-break:normal;}
+  .tg th{border-color:black;border-style:solid;border-width:1px;font-family:Arial, sans-serif;font-size:14px;
+    font-weight:normal;overflow:hidden;padding:10px 5px;word-break:normal;}
+  .tg .tg-c3ow{border-color:black;text-align:center;vertical-align:top}
+  .tg .tg-0pky{border-color:black;text-align:left;vertical-align:top}
+  .tg .tg-dgl5{border-color:black;background-color:#FFF;text-align:left;vertical-align:top}
+  </style>
+  <table class="tg">
+  <thead>
+    <tr>
+      <th class="tg-c3ow" colspan="2"><span style="font-weight:bold">magic-modules - build commands</span></th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td class="tg-0pky"><span style="font-weight:bold">make</span><br>runs make build</td>
```

Comment:
`make`, `make build`, `make terraform` are all aliases; we should choose one and only document that one to make things easier / clearer for users who are just getting started. My initial thought would be `make terraform` because it's unambiguous - but it's also from a time when we were supporting multiple downstreams that weren't necessarily terraform-related. We could instead do `make provider` or similar. What do you think?